src/main/java/com/google/api/codegen/gapic/ProtoModels.java

```diff
@@ -27,8 +29,19 @@
 
   private ProtoModels() {}
 
+  /** Gets the interfaces for the apis in the model. */
+  public static List<Interface> getInterfaces(
+      Model model, @Nullable GapicProductConfig productConfig) {
+    if (productConfig == null || !productConfig.getProtoParser().isProtoAnnotationsEnabled()) {
+      return getInterfacesFromServiceConfig(model);
+    }
+
+    // Assume valid service config if proto annotations parsing is not enabled.
```

Comment:
Won't we end up here if proto annotation parsing *is* enabled?

Reply:
Oh yes, fixed comment.